### ISSUE TITLE
Make parse_gas! return u64

### ISSUE THREAD

**Issue by flmel** [near/units-rs#10]
Maybe u64 instead of u128 would be better in this case, even better would be to return Gas type

```rust
Self::ext(env::current_account_id())
    .with_static_gas(Gas(parse_gas!("5 T") as u64)) // 5*10e12, 5Tgas
    .save_contract_callback(ft_contract_id),
```

**Comment by willemneal** [comment]
Indeed! I wanted to do this but was removed from the org before I could do it. I can help review if you want to make a PR. Then perhaps the current maintainers will approve it.

**Comment by frol** [comment]
@flmel Please, use [near-gas](https://crates.io/crates/near-gas) crate instead of near-units.
